deps/ncrypto/ncrypto.h

```diff
@@ -1539,6 +1539,21 @@ DataPointer pbkdf2(const Digest& md,
                    uint32_t iterations,
                    size_t length);
 
+#ifndef OPENSSL_NO_ARGON2
```

Comment:
Do you need the `#if OPENSSL_VERSION_MAJOR >= 3 && OPENSSL_VERSION_MINOR >= 2` here too?

Reply:
Yes, otherwise there will be compilation errors.